index.html

```diff
@@ -299,32 +299,37 @@
 
 
       function translate(node, info) {
-        if((node.nodeType === 1 && node.hasAttribute('data-ly-locked')) || (node.nodeType === 3 && node.parentNode && node.parentNode.hasAttribute('data-ly-locked'))) {
+        const nodeType = node.nodeType;
+        const nodeName = node.nodeName;
+
+        if ((nodeType === 1 && node.hasAttribute('data-ly-locked')) || (nodeType === 3 && node.parentNode && node.parentNode.hasAttribute('data-ly-locked'))) {
           return;
         }
+
+        if (nodeType === 1 && node.style.backgroundImage) {
+          translateCssImage(node);
+        }
```

Comment:
I moved the background check here so we can have the `if...else if` part below. In my testing it seems like not only `node.style.backgroundImage` is faster than `node.getAttribute('style')` but it also allows us to skip translating all of the nodes that have the `style` attribute but don't have the background image set.

That said, `translateCssImage` still has to parse and validate the attribute, because the background image can be set by inline/external CSS and not the `style` attribute.